lib/datadog/tracing/contrib/http/configuration/settings.rb

```diff
@@ -33,6 +33,8 @@ class Settings < Contrib::Configuration::Settings
               o.lazy
             end
 
+            option :response_code_errors, default: 400...599
```

Comment:
Thinking maybe this should be configurable by env var, too? Not all users can practically configure using code in their deployments.

@marcotc thoughts?

Reply:
I went ahead and added this as well so it could be reviewed, happy to revert that change if it would be better not to add it